app/models/ai_persona.rb

```diff
@@ -171,6 +232,25 @@ def class_instance
     end
   end
 
+  def create_user!
+    raise "User already exists" if user_id && User.exists?(user_id)
+
+    # note .invalid is a reserved TLD which will route nowhere
+    user =
+      User.new(
+        email: "#{SecureRandom.hex}@does-not-exist.invalid",
```

Comment:
We do this in a few places in core, would be cool to have a `User.generate_fake_email` method in core. No need to change here, just saying.

Reply:
yeah agree... the secure random hex is such a tricky thing to come up with and it is non obvious... plus I needed to wikipedia to find out about `.invalid`

Reply:
Yeah it's not obvious what `.invalid` does unless you've already had to deal with it